### DISCUSSION TITLE
Access the latLng property of the google.maps.MapMouseEvent interface

### DISCUSSION THREAD

**Discussion by smissaertj** [inocan-group/vue3-google-map#231] [Q&A]
When I click on the map, I want to capture the underlying coordinates to set a Marker at that position later on. It's not clear to me how can I access the `latLng` property of the `google.maps.MapMouseEvent` interface?

https://developers.google.com/maps/documentation/javascript/reference/map#MapMouseEvent

Example here:
https://developers.google.com/maps/documentation/javascript/examples/event-click-latlng

**Comment by HusamElbashir** [comment]
Hi

Please refer to https://github.com/inocan-group/vue3-google-map/issues/91#issuecomment-1179765783
